shell.py

```diff
@@ -311,6 +311,33 @@ def help_ls(self):
         self.stdout.write('List directory contents.\n' +
                           'Use ls -a to show hidden files\n')
 
+def help_cp(self):
+    self.stdout.write('Copies contents of one text file to a new text file\n' +
+                        'syntax: cp source.py destination.py\n')
```

Comment:
I would update this to not use .py explicitly and just say `cp source dest` and remove the word 'text'